codalab/lib/bundle_store.py

```diff
@@ -383,3 +395,76 @@ def cleanup(self, uuid, dry_run):
         if not dry_run:
             path_util.remove(absolute_path)
 
+
+    def health_check(self, model):
+        """
+        MultiDiskBundleStore.health_check(): In the MultiDiskBundleStore, bundle contents are stored on disk, and
+        occasionally the disk gets out of sync with the database, in which case we make repairs in the following ways:
+
+            1. Creates a trash directory
+            2. Moves the contents of bundles with corresponding UUID not in the database to trash
+            3. Any files that don't begin with a UUID string should be moved to trash
+            4. For each bundle marked READY or FAILED, ensure that its dependencies are not located in the bundle
+               directory.
+            5. Bundles that are stored in <UUID>.cid or <UUID>.status/ should be marked as READY or FAILED
+            6. <UUID>.sh and <UUID>-internal.sh should be gone for bundles marked READY or FAILED
+        """
+        # Create a trash directory
+        trash_dir = os.path.join(self.codalab_home, 'trash')
```

Comment:
Let's not move things into a trash directory because this might involve transferring between disks.  Instead, prints out 'rm -r ...' on the things that should be deleted it, and have a force mode that actually executes these commands.